docs/plugins/subsonicupdate.rst

```diff
@@ -13,11 +13,11 @@ You can do that using a ``subsonic:`` section in your ``config.yaml``,
 which looks like this::
 
     subsonic:
-        host: X.X.X.X
-        port: 4040
+        url: https://example.com:443/subsonic
         user: username
         pass: password
-        contextpath: /subsonic
+
+\* NOTE: The pass config option can either be clear text or hex-encoded with a "enc:" prefix.
```

Comment:
http://www.subsonic.org/pages/api.jsp from there API docs

Reply:
Cool! Buy maybe this sentence should go below, in the bullet for the `pass` configuration option, rather than by itself here.